src/components/PageContent/PageContent.astro

```diff
@@ -2,8 +2,9 @@
 import MoreMenu from '../RightSidebar/MoreMenu.astro';
 import ArticleNavigationButton from './ArticleNavigationButton.astro';
 import TableOfContents from '../RightSidebar/TableOfContents.tsx';
-import { getLanguageFromURL } from '../../util.ts';
-import { SIDEBAR } from '../../config.ts';
+import { getUIString } from "~/i18n/util.ts";
```

Comment:
Nit: the rest of this codebase uses relative URL imports and not aliases, for consistency can you match?